src/Symfony/Component/ObjectMapper/Tests/Fixtures/ServiceLocator/TransformCallable.php

```diff
@@ -18,7 +18,7 @@
  */
 class TransformCallable implements TransformCallableInterface
 {
-    public function __invoke(mixed $value, object $object): mixed
+    public function __invoke(mixed $value, object $source, ?object $target = null): mixed
```

Comment:
```suggestion
    public function __invoke(mixed $value, object $source, ?object $target): mixed
```
maybe? Same for `src/Symfony/Component/ObjectMapper/Tests/Fixtures/ServiceLocator/ConditionCallable.php`